server/test/homeRoute.test.js

```diff
@@ -0,0 +1,75 @@
+const request = require('supertest');
+const app = require('../app');
+
```

Comment:
I suppose that you run build for test_url,
But this is locally
So we need here to run the build function 
```js
beforeAll(() => build)
```
and after all we need to close connection